packages/bot-web-ui/src/pages/dashboard/dashboard.tsx

```diff
@@ -28,17 +28,19 @@ const DashboardComponent = observer(({ handleTabChange }: TMobileIconGuide) => {
         if (!get_first_strategy_info.current) {
             //on dashbord umount fire close event for rudderstack
             get_first_strategy_info.current = true;
-            Analytics.trackEvent('ce_bot_dashboard_form', {
-                action: 'open',
-                form_source: 'ce_bot_dashboard_form',
+            rudderstackDashboardOpen({
+                bot_name: dashboard_strategies?.[0]?.name,
+                preview_mode: dashboard_strategies.length ? 'yes' : 'no',
```

Comment:
@farabi-deriv Bhai can we put a null check on dashboard_strategies.length? Just to ensure if the value is null then it's not affecting the App.
```suggestion
                preview_mode: dashboard_strategies?.length ? 'yes' : 'no',
```

Reply:
Sure @shafin-deriv